javaext/com.microsoft.java.lombok/src/main/java/com/microsoft/java/lombok/ConstructorHandler.java

```diff
@@ -0,0 +1,101 @@
+package com.microsoft.java.lombok;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+
+import org.eclipse.jdt.core.IType;
+import org.eclipse.jdt.core.dom.IMethodBinding;
+import org.eclipse.jdt.core.dom.ITypeBinding;
+import org.eclipse.jdt.core.dom.ASTNode;
+import org.eclipse.jdt.core.dom.CompilationUnit;
+import org.eclipse.jdt.core.manipulation.CoreASTProvider;
+import org.eclipse.jdt.internal.corext.dom.ASTNodes;
+import org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin;
+import org.eclipse.jdt.ls.core.internal.preferences.Preferences;
+import org.eclipse.jdt.internal.corext.codemanipulation.CodeGenerationSettings;
+import org.eclipse.jdt.ls.core.internal.handlers.GenerateConstructorsHandler;
+import org.eclipse.jdt.ls.core.internal.handlers.GenerateConstructorsHandler.CheckConstructorsResponse;
+import org.eclipse.jdt.ls.core.internal.handlers.JdtDomModels.LspMethodBinding;
+import org.eclipse.jdt.ls.core.internal.handlers.JdtDomModels.LspVariableBinding;
+import org.eclipse.jdt.ls.core.internal.text.correction.SourceAssistProcessor;
+import org.eclipse.jdt.core.dom.rewrite.ListRewrite;
+import org.eclipse.lsp4j.CodeActionParams;
+import org.eclipse.text.edits.TextEdit;
+
+import com.microsoft.java.lombok.ConstructorHandler;
+
+public class ConstructorHandler {
+    public static void removeMethods(IType type, ListRewrite rewriter, ConstructorKind kind, IProgressMonitor monitor) {
+        try{
+            CompilationUnit astRoot = CoreASTProvider.getInstance().getAST(type.getCompilationUnit(), CoreASTProvider.WAIT_YES, monitor);
+            if (astRoot == null) {
+		    	return;
+            }
+            ITypeBinding typeBinding = ASTNodes.getTypeBinding(astRoot, type);
+		    if (typeBinding == null) {
```

Comment:
looks like your code is using a mix of spaces and tabs when indenting. It's better to use one kind of style for indenting.